stix2/equivalence/graph/__init__.py

```diff
@@ -95,88 +100,36 @@ def graphically_equivalent(ds1, ds2, prop_scores={}, **weight_dict):
 
 
 # default weights used for the graph semantic equivalence process
-# values are re-balanced to account for new property checks and add up to 100
-WEIGHTS = {
-    "attack-pattern": {
-        "name": (30, partial_string_based),
-        "external_references": (70, partial_external_reference_based),
-    },
-    "campaign": {
-        "name": (60, partial_string_based),
-        "aliases": (40, partial_list_based),
-    },
-    "course-of-action": {
-        "name": (60, partial_string_based),
-        "external_references": (40, partial_external_reference_based),
-    },
-    "identity": {
-        "name": (60, partial_string_based),
-        "identity_class": (20, exact_match),
-        "sectors": (20, partial_list_based),
-    },
-    "indicator": {
-        "indicator_types": (15, partial_list_based),
-        "pattern": (80, custom_pattern_based),
-        "valid_from": (5, partial_timestamp_based),
-        "tdelta": 1,  # One day interval
-    },
-    "intrusion-set": {
+WEIGHTS.update({
```

Comment:
Should we copy `WEIGHTS` and update/use the copy instead? That way if someone performs graph equivalence and then later performs object equivalence it won't do the additional stuff that graph equivalence does.

Reply:
I think the fact that they also shared the same name caused a problem in scope resolution. I renamed the graph one...